SimpleKeychain/A0SimpleKeychain.m

```diff
@@ -356,12 +356,12 @@ - (NSDictionary *)queryFetchOneByKey:(NSString *)key message:(NSString *)message
                                       (__bridge id)kSecAttrAccount: key,
                                       }];
 #if TARGET_OS_IPHONE
-    if (self.useAccessControl && floor(NSFoundationVersionNumber) > NSFoundationVersionNumber_iOS_7_1) {
-        if (message) {
+    if (self.useAccessControl) {
+        if (message && floor(NSFoundationVersionNumber) > NSFoundationVersionNumber_iOS_7_1) {
             query[(__bridge id)kSecUseOperationPrompt] = message;
+        } else {
+            NSLog(@"Error trying to access to non available kSecUseOperationPrompt in iOS7");
```

Comment:
If you want to remove the log message then I'll go back to the same format as before and just remove the log message - this was just so it didn't print a misleading error message when `useAccessControl` was false. Let me know what you'd like, thanks!